tools/SourceKit/CMakeLists.txt

```diff
@@ -4,6 +4,29 @@ include(CheckSymbolExists)
 list(APPEND CMAKE_MODULE_PATH
     "${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules")
 
+if(NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
```

Comment:
@akyrtzi does SourceKit still support being built out of tree? If not, maybe we just put in an assert here and get rid of the if statement?

Reply:
There's no reason to build it out of tree, removing SGTM.